source/user-manual/capabilities/vulnerability-detection/cpe-helper.rst

```diff
@@ -586,3 +598,207 @@ To illustrate the process, we will follow the same procedure as in the previous
             "reference":"http://www.securityfocus.com/bid/106928"
         }
     }
+
+Products with version update field
+----------------------------------
+
+Another use case where vulnerabilities cannot be confirmed by the NVD due to the generic version contained in the CPE is,
+for example, the vulnerability *CVE-2022-23277* for the *Microsoft Exchange Server* software product,
+which also has a new field with the ``update`` version, where the cumulative version of the product is specified.
+
+If we consult this product in the `National Vulnerability Database (CVE-2022-23277) <https://nvd.nist.gov/vuln/detail/CVE-2022-23277>`_,
+we will find the following CPEs:
+
+
+.. thumbnail:: ../../../images/manual/vuln-detector/nvd-exchange-server.png
+    :title: Affected software for CVE-2022-23277
+    :align: center
+    :width: 100%
+
+Where we can observe, that in the CPEs the version and update fields are specified with generic versions that do not allow us
```

Comment:
I'd try to reorganize this sentence to make it sound less colloquial. In addition, I don't understand the `generic versions` meaning, they seem specific versions.